### PR TITLE
Update Windows CLI install instructions

### PR DESCRIPTION
**Pull request by jwitz**
It looks like Winget support is still some time away, so let's document the `exe` setup in the meantime

### CODE REVIEW

**Review by neel-astro — APPROVED** [review]

File: astro/cli/install-cli.md [hunk 1]
```diff
@@ -61,17 +61,13 @@ This is where you'll find information about installing the Astro CLI on a Window
...
 #### Installation
 
-1. Go to the [Releases page](https://github.com/astronomer/astro-cli/releases) of the Astro CLI GitHub repository, scroll to a CLI version, and then download the `.zip` file that matches the CPU architecture of your machine.
+1. Go to the [Releases page](https://github.com/astronomer/astro-cli/releases) of the Astro CLI GitHub repository, scroll to a CLI version, and then download the `.exe` file that matches the CPU architecture of your machine.
 
-    For example, to install v1.0.0 of the Astro CLI on a Windows machine with an AMD 64 architecture, download `astro_1.0.0-converged_windows_amd64.zip`.
+    For example, to install v1.0.0 of the Astro CLI on a Windows machine with an AMD 64 architecture, download `astro_1.0.0-converged_windows_amd64.exe`.
 
-2. If the `.zip` file isn't automatically extracted, run the following command to extract the executable:
+2. Rename the file to `astro.exe`.
```
Comment (kwcanuck) [inline astro/cli/install-cli.md:68]:
```suggestion
2. Rename the file `astro.exe`.
```

File: astro/cli/install-cli.md [hunk 2]
```diff
@@ -130,15 +126,11 @@ brew install astro
...
-     For example, to upgrade to v1.0.0 of the Astro CLI on a Windows machine with an AMD 64 architecture, you download `astro_1.0.0-converged_windows_amd64.zip`.
+2. Go to the [Releases page](https://github.com/astronomer/astro-cli/releases) of the Astro CLI GitHub repository, scroll to a CLI version, and then download the `.exe` file that matches the CPU architecture of your machine.
 
-3. If the `.zip` file isn't automatically extracted, run the following command to extract the executable:
+     For example, to upgrade to v1.0.0 of the Astro CLI on a Windows machine with an AMD 64 architecture, you download `astro_1.0.0-converged_windows_amd64.exe`.
 
-    ```sh
-    tar -xvzf .\astrocli.tar.gz
-    ```
+3. Rename the file to `astro.exe`.
```
Comment (kwcanuck) [inline astro/cli/install-cli.md:133]:
```suggestion
3. Rename the file `astro.exe`.
```

File: astro/cli/install-cli.md [hunk 3]
```diff
@@ -61,17 +61,13 @@ This is where you'll find information about installing the Astro CLI on a Window
...
 
-2. If the `.zip` file isn't automatically extracted, run the following command to extract the executable:
+2. Rename the file to `astro.exe`.
 
-    ```sh
-    tar -xvzf .\astrocli.tar.gz
-    ```
-
-3. Add the filepath for the directory containing `astro.exe` as a PATH environment variable. For example, if `astro.exe` was stored in `C:\Users\username\astro.exe`, you would add `C:\Users\username` as your PATH environment variable. To learn more about configuring the PATH environment variable, see [Java documentation](https://www.java.com/en/download/help/path.html).
+3. Add the filepath for the directory containing the new `astro.exe` as a PATH environment variable. For example, if `astro.exe` was stored in `C:\Users\username\astro.exe`, you would add `C:\Users\username` as your PATH environment variable. To learn more about configuring the PATH environment variable, see [Java documentation](https://www.java.com/en/download/help/path.html).
```
Comment (kwcanuck) [inline astro/cli/install-cli.md:70]:
```suggestion
3. Add the filepath for the directory containing the new `astro.exe` as a PATH environment variable. For example, if `astro.exe` is stored in `C:\Users\username\astro.exe`, you add `C:\Users\username` as your PATH environment variable. To learn more about configuring the PATH environment variable, see [How do I set or change the PATH system variable?](https://www.java.com/en/download/help/path.html).
```

File: astro/cli/install-cli.md [hunk 4]
```diff
@@ -130,15 +126,11 @@ brew install astro
...
 
-3. If the `.zip` file isn't automatically extracted, run the following command to extract the executable:
+     For example, to upgrade to v1.0.0 of the Astro CLI on a Windows machine with an AMD 64 architecture, you download `astro_1.0.0-converged_windows_amd64.exe`.
 
-    ```sh
-    tar -xvzf .\astrocli.tar.gz
-    ```
+3. Rename the file to `astro.exe`.
 
 4. Add the filepath for the directory containing the new `astro.exe` as a PATH environment variable. For example, if `astro.exe` was stored in `C:\Users\username\astro.exe`, you would add `C:\Users\username` as your PATH environment variable. To learn more about configuring the PATH environment variable, see [Java documentation](https://www.java.com/en/download/help/path.html).
```
Comment (kwcanuck) [inline astro/cli/install-cli.md:135]:
```suggestion
4. Add the filepath for the directory containing the new `astro.exe` as a PATH environment variable. For example, if `astro.exe` is stored in `C:\Users\username\astro.exe`, you add `C:\Users\username` as your PATH environment variable. To learn more about configuring the PATH environment variable, see [How do I set or change the PATH system variable?](https://www.java.com/en/download/help/path.html).
```

File: software/install-cli.md [hunk 5]
```diff
@@ -60,17 +60,13 @@ To use the Astro CLI on Windows, you must have:
...
 
-2. If the `.zip` file does not automatically unzip, run the following command to unzip the executable:
+2. Rename the file to `astro.exe`.
 
-    ```sh
-    tar -xvzf .\astrocli.tar.gz
-    ```
-
-3. Add the filepath for the directory containing `astro.exe` as a PATH environment variable. For example, if `astro.exe` was stored in `C:\Users\username\astro.exe`, you would add `C:\Users\username` as your PATH environment variable. To learn more about configuring the PATH environment variable, see [Java documentation](https://www.java.com/en/download/help/path.html).
+3. Add the filepath for the directory containing the new `astro.exe` as a PATH environment variable. For example, if `astro.exe` was stored in `C:\Users\username\astro.exe`, you would add `C:\Users\username` as your PATH environment variable. To learn more about configuring the PATH environment variable, see [Java documentation](https://www.java.com/en/download/help/path.html).
```
Comment (kwcanuck) [inline software/install-cli.md:69]:
```suggestion
3. Add the filepath for the directory containing the new `astro.exe` as a PATH environment variable. For example, if `astro.exe` is stored in `C:\Users\username\astro.exe`, you add `C:\Users\username` as your PATH environment variable. To learn more about configuring the PATH environment variable, see [How do I set or change the PATH system variable?](https://www.java.com/en/download/help/path.html).
```

**Review by kwcanuck — APPROVED** [review]

File: astro/cli/install-cli.md (hunk 1 above)
Comment (paolaperaza) [reply astro/cli/install-cli.md:68]: I like the "to" here? I can't tell if this was saying "rename the file called this" or "rename the file TO this name"

**Review by paolaperaza — APPROVED** [review]

File: astro/cli/install-cli.md (hunk 1 above)
Comment (kwcanuck) [reply astro/cli/install-cli.md:68]: The addition of the preposition _to_ is unnecessary here. If you don't like the existing sentence, you'll need to up[date it to:

Rename `.exe` to `astro.exe`.

Currently, the transitive verb _rename_ has an object and it's assumed the reader will remember what file is being renamed as it was referenced in step 1.

**Pull request merged**